Revit_Engine/Modify/SetPullEdges.cs

```diff
@@ -36,18 +36,17 @@ public static partial class Modify
         [Description("Sets Pull Edges option for FilterQuery.")]
         [Input("filterQuery", "FilterQuery")]
         [Input("pullEdges", "Set to true if include geometry edges of Revit Element in CustomData of BHoMObject")]
+        [Input("includeNonVisibleObjects", "Set to true if include non visible objects of Revit Element in geometry")]
         [Output("FilterQuery")]
-        public static FilterQuery SetPullEdges(this FilterQuery filterQuery, bool pullEdges)
+        public static FilterQuery SetPullEdges(this FilterQuery filterQuery, bool pullEdges = true, bool includeNonVisibleObjects = false)
```

Comment:
One comment here regarding backwards compatibility for old scripts.

When modifying a public method like this, by adding inputs, please keep the old method as well, and make it call this one with the default attributes. Also put a `Deprecated` tag on it. This is to avoid breaking existing scripts making use of this method. [see here](https://github.com/BHoM/documentation/wiki/Backwards-compatibility#methodscomponents)

This would be adding 

```
        [Deprecated("2.2")]
        public static FilterQuery SetPullEdges(this FilterQuery filterQuery, bool pullEdges = true)
        {
            return SetPullEdges(filterQuery, pullEdges, false);
        }
```

To what is already in the file.